### ISSUE TITLE
Problem with Visual sync and VLC Media Player

### ISSUE THREAD

**Issue by AllanHigdon** [SubtitleEdit/subtitleedit#8185]
I have a problem when I use the Visual sync feature along with VLC 3.0.20.
Absolutely nothing happens when I select the Play button for the end scene.
The Play button for the start scene works fine.
The last version that worked correctly for me was v4.0.1.
Can someone else confirm this problem?

**Comment by niksedk** [comment]
Sorry, I cannot re-create this.
But do try mpv as video player in SE which is a lot better.

**Comment by budryerson** [comment]
I have the same problem: `Synchronization > Visual sync > End scene > Play/Pause icon`
When using **VLC Media Player**, clicking on the `End scene > Play/Pause icon` does nothing.

I first noticed when I upgraded **SubtitleEdit** from `v3.x.x` to `v4.0.10`.  I tried downgrading back to `v4.0.9`, but there was no change. I tried upgrading VLC from `v3.0.18` to the latest version, `v3.0.20`, but there was still no change.

Clicking on the media window `Starts` and `Stops` the media. The button icon changes, too.
`End scene > Play/Pause icon` does not work with VLC.
